src/internal/observable/dom/AjaxObservable.ts

```diff
@@ -344,7 +351,7 @@ export class AjaxSubscriber<T> extends Subscriber<Event> {
           const { progressSubscriber } = (<any>xhrProgress);
           progressSubscriber.next(e);
         };
-        if (root.XDomainRequest) {
+        if (XDomainRequest) {
```

Comment:
will this work node.js uses polyfills? it seems like node 14 at least doesn't seem to like it 
![image](https://user-images.githubusercontent.com/1210596/86617097-9c207480-bf6b-11ea-95d1-dc0856625e9b.png)

we've been guiding user to use polyfills on node.js for ajaxobservable, I believe numbers will be quite small though.

Reply:
If the polyfill puts something on `global`, it _should_ work. I suppose I could make it a little less volatile though